cpp/src/sort/sort.cuh

```diff
@@ -0,0 +1,70 @@
+// -*-c++-*-
+
+#pragma once
+
+#include <cudf/types.h>
+
+namespace cusort {
+
+  /**
+   * @brief  Sort key value pairs distributed across multiple GPUs
+   *
+   *   This sort function takes arrays of keys and values distributed
+   *   around multiple GPUs 
+   *
+   * @param[in]   input_keys_d     The unsorted keys, stored in device arrays.
+   *                               input_keys_d[i] is the array of keys on GPU i
+   * @param[in]   input_values_d   The unsorted values, stored in device arrays.
+   *                               input_values_d[i] is the array of values on GPU i
+   * @param[in]   input_length_h   Host array containing the number of elements on
+   *                               each GPU in the input key/value arrays.
+   * @param[out]  output_keys_d    The sorted keys, stored in device arrays.
+   *                               output_keys_d[i] is the array of keys on GPU i
+   * @param[out]  output_values_d  The sorted values, stored in device arrays.
+   *                               output_values_d[i] is the array of values on GPU i
+   * @param[out]  output_length_h  Host array containing the number of elements on
+   *                               each GPU in the output key/value arrays.
+   * @param[in]   num_gpus         The number of GPUs
+   *
```

Comment:
AFAIK, using h_ for host variables and d_ for device variables is more common than using _h and _d.

Reply:
OK, I'll change these.